gordo_components/cli/workflow_generator.py

```diff
@@ -135,6 +142,13 @@ def workflow_generator_cli(**ctx: dict):
         context.pop("n_servers") or len(config.machines) * 10
     )
 
+    log_level = config.globals["runtime"].get("log_level", "DEBUG")
+    context["log_level"] = log_level
```

Comment:
This will overwrite the log-level provided in the CLI

Reply:
Also, remember to set the loglevel of gordo-deploy itself to the provided loglevel...